src/HonzaBotner.Discord.Services/Commands/MessageCommands.cs

```diff
@@ -219,5 +220,50 @@ public async Task Dump(CommandContext ctx, [Description("URL of the message.")]
                 await ctx.Message.RespondAsync($"TODO: {roles.Count}");
             }*/
         }
+
+        [Group("buttons")]
+        [Description("Module used to edit button interactions on messages")]
+        [ModuleLifespan(ModuleLifespan.Transient)]
+        public class ButtonCommands : BaseCommandModule
```

Comment:
I guess  this should be annotated with `[RequireMod]` attribute

Reply:
Ups 😅 I thought it's enough when the main method is RequireMod, as is Message here. But I haven't found any documentation about it. Will fix later.

Reply:
Checked - It's hidden behind that attribute and works as intended.